Sentiment_Analysis/README.md

```diff
@@ -0,0 +1,29 @@
+#Sentiment Analysis#
```

Comment:
Why do we have  duplicates of dictionaries_review documents? which one is the final one? I'd remove the intermediate steps and rely on the version control.